blog/views.py

```diff
@@ -21,18 +21,22 @@ class PostForm(forms.Form):
         (Privacy.FRIENDS, "Friends only"),
         (Privacy.FOAF, "Friends of friends"),
     )))
+    image = forms.ImageField(label = 'Choose an Image')
     content_type = forms.CharField(widget=forms.Select(choices=(
         ("text/plain", "Plain Text"),
         ("text/markdown", "Markdown"),
     )))
 
+
+
+
 def post(request):
     author = users.models.Author.from_user(request.user)
     if author is None:
         return redirect("login")
 
     if request.method == "POST":
-        form = PostForm(request.POST)
+        form = PostForm(request.POST,request.FILES)
 
         if form.is_valid():
```

Comment:
currently when I submit a form with an image, the image is cleared, and it says field required, and I can't post